src/components/InputNotice/InputNotice.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components/macro';
+
+export const InputNotice = ({ error, errorText, helpText }) => {
+    return (
+        <Wrapper error={error}>
```

Comment:
Instead of hiding the Element in CSS, you could stop rendering this component at all. Two options:

```suggestion
export const InputNotice = ({ error, errorText, helpText }) => {
  if (error) {
    return null
  }
    return (
        <Wrapper>
```

or even better:

```jsx
{error && <InputNotice
                error={error}
                errorText={errorText}
                helpText={helpText}
            />}
```